tests/test_migrate.py

```
@@ -0,0 +1,1417 @@
+    migrate_plugins,
+    migrate_store,
+    perform_migration,
+    rollback_migration,
+    verify_migration,
+)
+
+
+class TestPathIsWithinHome:
+    """Tests for _path_is_within_home function (lines 72-84)."""
+
+    def test_path_within_home(self, clean_migration_home, tmp_path):
```

Ruff (0.14.14)

[warning] Unused method argument: `tmp_path`

(ARG002)